test/e2e/node/security_context.go

```diff
@@ -146,6 +179,88 @@ var _ = SIGDescribe("Security Context", func() {
 		testPodSELinuxLabeling(f, false, true)
 	})
 
+	addSeccompProfilesToNode := func(profiles []*seccompProfile) {
+		name := "seccomp-setup-" + string(uuid.NewUUID())
+		hostPathType := v1.HostPathDirectoryOrCreate
+		seccompProfileRoot := filepath.Join(services.KubeletRootDirectory, "seccomp")
+
+		args := []string{}
+		for _, profile := range profiles {
+			str, err := profile.toEchoStr(seccompProfileRoot)
+			framework.ExpectNoError(err)
+			args = append(args, str)
+		}
+
+		ds := &appsv1.DaemonSet{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:   name,
+				Labels: map[string]string{"name": name},
+			},
+			Spec: appsv1.DaemonSetSpec{
+				Selector: &metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						"name": "seccomp-setup",
+					},
+				},
+				Template: v1.PodTemplateSpec{
+					ObjectMeta: metav1.ObjectMeta{
+						Labels: map[string]string{"name": "seccomp-setup"},
+					},
+					Spec: v1.PodSpec{
+						Containers: []v1.Container{
+							{
+								Name:  "test-container",
+								Image: imageutils.GetE2EImage(imageutils.BusyBox),
+								VolumeMounts: []v1.VolumeMount{
+									{Name: "kubelet", MountPath: seccompProfileRoot},
+								},
+								Command: []string{
+									"sh", "-c", strings.Join(args, " && ") + " && sleep 6000",
+								},
+							},
+						},
+						Volumes: []v1.Volume{
+							{
+								Name: "kubelet",
+								VolumeSource: v1.VolumeSource{
+									HostPath: &v1.HostPathVolumeSource{
+										Path: seccompProfileRoot,
+										Type: &hostPathType,
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		}
+
+		ns := f.Namespace.Name
+		_, err := f.ClientSet.AppsV1().DaemonSets(ns).Create(
+			context.Background(), ds, metav1.CreateOptions{},
+		)
+		framework.ExpectNoError(err)
+		framework.ExpectNoError(e2epod.WaitForPodsRunningReady(
+			f.ClientSet, ns, 1, 0, framework.PollShortTimeout, nil,
+		))
+	}
+
+	ginkgo.It("should support seccomp localhost profile for a container [LinuxOnly]", func() {
```

Comment:
the test below has a `[Feature:Seccomp]` tag which seems appropriate, nothing actually requires linux hosts to have seccomp enabled, right ...?

Reply:
That’s right. Our plan is to remove the tag from all tests when graduating seccomp to GA in 1.19. I decided to not add it in the first place to not encounter to many merge conflicts. Do you think that is fine?

Reply:
> I need to get more time to look at this -- how does this behave if the host does not have selinux enabled / setenforce?

I'm not aware of any interactions between seccomp and selinux. I think they're two distinct features. Maybe @rhatdan can help us here?

Reply:
SECCOMP and SELinux do not rely on each other. They can both be enabled/disabled independent of each other.